nanomsg.sh

```diff
@@ -1,13 +1,34 @@
 package: nanomsg
-version: v0.8
+version: v1.0.0
 source: https://github.com/nanomsg/nanomsg
-tag: 0.8-beta
+tag: 1.0.0
 build_requires:
-  - autotools
+  - CMake
 ---
 #!/bin/sh
 rsync -a --delete --exclude '**/.git' $SOURCEDIR/ ./
```

Comment:
Sorry for taking long to review this. cmake should be able to build out of sources. Can you please check this is the case and use something like:

```
cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX:PATH="${INSTALLROOT}"
```

and then drop the rsync?